app/builders/immediate_email_builder.rb

```diff
@@ -16,14 +16,14 @@ def records
     @records ||= begin
       now = Time.zone.now
       recipients_and_content.map do |recipient_and_content|
-        subscriber = recipient_and_content.fetch(:subscriber)
         content = recipient_and_content.fetch(:content)
-        subscriptions = recipient_and_content.fetch(:subscriptions)
+        subscription = recipient_and_content.fetch(:subscription)
```

Comment:
You've got a minor typo on the commit message for this.

Reply:
I assume you mean `and` (-> `an`). Fixed.